### DISCUSSION TITLE
Cant connect to wifi. Not sure what I'm doing wrong

### DISCUSSION THREAD

**Discussion by trevor-gerring** [plmilord/Hass.io-custom-component-ikamand#10] [General]
I'm fairly new to HACS and Home Assistant. I followed the instructions, but it appears that the iKammand is unable to connect to my wifi. I press and hold the button on the side, get the flashing blue light, go to http://192.168.10.1/cgi-bin/netset, enter my wifi details in Base64, and hit submit. I get a screen that says:
```
 XXXXXXXXXXXX 
Content-type: text/html
Status: 200/html

<html>
<head>
<title>Network Status</title>
<link rel="stylesheet" type="text/css" href="/style.css">
</head>
<body bgcolor="#fffeec" text="black">
<br>
</div>
<div class="contentblock">
<dl>

<form name="netset" action="/cgi-bin/netset" method="post">
<p id="SSID"> SSID: </p><input type="text" name="ssid"/>
<p id="PW"> PASSWORD: </p><input type="password" name="pass"/>
<p id="USER"> USERID: </p><input type="text" name="user"/>
<input type="submit" onclick="post()"/>
</form>
<p id ="mac-address">MAC ADDR: XXXXXXXXXXXX </p></dl>
</body>
</html>
```
Is this normal/ what it's supposed to do?

The iKammand appears to restart after hitting submit but then just has an intermittent blinking blue light, which I think means it's trying to connect to a network (solid blue is connected?). It also never shows up in my router connected devices list.

I'm using an eero router. I originally thought it was because it was getting caught up with the 5GHz bands, so I limited my network to 2.4GHz thinking that might help it connect, no luck.

Any help or tips on what I'm doing wrong would be appreciated.

**Comment by plmilord** [comment]
This seems weird... have you overcome this problem since this post?

Have your “SSID” and “Password” been base64 encoded?

When connecting to your router, do you see your iKamand connected to your 2.4Ghz WiFi?

**Reply by trevor-gerring** [reply]
Yes SSID and password in base64. iKamand never shows up in my list of connected devices. Still a mystery. 

Do others get the above screen/ code when submitting the form?

**Reply by plmilord** [reply]
Do you have two separate SSIDs (i.e. one at 2.4GHz and another at 5GHz)?

**Reply by trevor-gerring** [reply]
I don't, and Eero doesn't support this. You can temporarily disable the 5GHz network for setting up 2.4GHz devices, which I did, but I still couldn't connect.

**Reply by RichardMajer01** [reply]
I have the same issue. 
After searching info on Web I understood that SSID and password should have only small letters and numbers. I did wifi hotspot on my mobile with very simple SSID and password (majerovi) and used base64 encode. 
Unfortunately device is not connecting. (2,4Ghz)
I have tried several combinations but w/o success.

**Reply by RichardMajer01** [reply]
Hallo, any idea how to connect ikamand to the network, please? 
2,4GHz network, very simple name and password without aby symbols, just letters, used base64 encode
Tried with hotspot created on mobile, used few different routers and AP. 
After sending name and password, ikamand is slowly flashing blue color LED and is not visible wifi network from ikamand
